reference/shaders-msl/asm/comp/relaxed-block-layout.asm.comp

```diff
@@ -7,6 +7,9 @@ struct foo
 {
     uint bar;
     packed_float3 baz;
+    uchar quux;
+    packed_uchar4 blah;
+    packed_half2 wibble;
```

Comment:
I don't see any code trying to read from these types. We need to verify it casts to unpacked types correctly.